README.md

```diff
@@ -2,24 +2,214 @@
 
 # PMIx Standard
 
-This repository contains the LaTeX source for the PMIx standard document, and discussions regarding the PMIx standard.
+PMIx is an application programming interface (API) specification to provide
+High Performance Computing (HPC) libraries and programming models with
+portable and well defined access to commonly available
+distributed computing system services.
 
-## Contributing to the PMIx Standard
+## Governance
 
-There are two main ways to contribute to the PMIx Standard: Participating in the discussion, and writing sections of the document.
+The PMIx Standard is governed by an Administrative Steering Committee (ASC) comprised of representatives from a consortium of research, academic, and industrial organizations.The ASC shall consist of one representative from each Member organization, as defined below, and shall perform the following functions:
 
-### Participating in the discussion
+* Define the mission of the PMIx Standard.
+* Plan release timelines for versions of the PMIx Standard based on available resources, target features, and target goals of the Member organizations.
+* Vote on all issues requiring community consensus (e.g., acceptance of proposed modifications to the Standard), one vote per Member organization. In addition to voting on proposed modifications, the ASC can be requested to vote on any issue submitted by a Member for consideration.
+* Assemble working groups to address specific areas of interest to the community.
+* Nominate and approve the appointment of individuals to fill the role of Release Manager (per release), as described below.
 
-If you have a question about the PMIx interface or want to raise a question for discussion then please file an Issue against this repository. We are using the Issues to discuss topics, and will link Pull Requests related to that topic to that Issue. This provides the community with a public, archived forum for discussion.
+The ASC is led by two Co-Chairs responsible for organizing and conducting the quarterly meetings. This includes:
 
-### Participatin in the writing
+* Announcing meeting agendas at least one month in advance of the meeting
```

Comment:
To this point, we probably need to capture the notion that PRs targeted for consideration at the meeting should be in a 'ready' state at 1 month out so that they can be identified and added to the agenda. That gives folks a month to review the PRs and prepare for the meeting.

Maybe something like this (it's rough)
> ASC Co-Chairs must send out an agenda 1 month before the meeting. Any PRs that wish to be considered for that meeting [must | should] be in the "Reading" (for Provisional class changes) or "Eligible" (for Stable class and Deprecation changes) state at this time to make it on the agenda. The ASC Co-Chairs may allow for PRs to be added to the agenda after this time on a case-by-case basis.

Reply:
I guess I thought that was communicated elsewhere where we talk about submitting a PR for consideration, but I can add a label to make it clearer.

Reply:
Here is what I added:

> * Announcing meeting agendas at least one month in advance of the meeting. Any proposal wishing to be considered for the next meeting must be put forward (by assigning the proper label, as described below) prior to that time. The ASC Co-Chairs may allow a proposal to be added to the agenda after this time on a case-by-case basis at their sole discretion.

Reply:
Looks good to me.